### DISCUSSION TITLE
Linked TAXII server page - HELP

### DISCUSSION THREAD

**Discussion by pdelv** [MISP/MISP#9003] [Q&A]
Hello everyone,
as a personal project, I'm trying to create a MISP <-> TAXII server connection to share feeds or information in STIX format.
I'm studying the TAXII world, and I noticed the discussion title page.
I think, this page would be used to connect directly with a TAXII server to PUSH or PULL data, but I couldn't find any documentation.
I tried to set up a new TAXII server, but I couldn't figure out what the two drop-box fields with the wrench are used for.
I guessed they would be populated after a correct server configuration, but, maybe I configured the server wrong, and nothing happened.
Can anyone give me any help understanding what this page is for?
Thanks a lot!

P.s. I also tried adding to the Baseurl field
/taxii/collections
/taxii/discovery
as suggested in many command-line taxii server operations, but still nothing

![Immagine 2023-04-06 171230](https://user-images.githubusercontent.com/119507780/230422934-47f246da-df5d-4e94-8f1f-39648a95b889.png)

**Comment by lululemon-harry** [comment]
same issues here and it looks like there is no guide or support on this issue :(
